app/src/main/java/com/developersbreach/composeactors/ui/screens/movieDetail/MovieDetailsUI.kt

```diff
@@ -68,6 +76,15 @@ fun MovieDetailsUI(
     val isLayerRevealAnimationEnded = rememberSaveable { mutableStateOf(false) }
     // Change button state with respect to scroll changes.
     val showFab = rememberSaveable { mutableStateOf(true) }
+    // Remember scroll state to change button state.
+    val showBottomSheetScaffold = rememberSaveable { mutableStateOf(true) }
```

Comment:
Should be shown only after `isLayerRevealAnimationEnded` is completed, right now Scaffold is just laid once user opens the screen and this isn't looking look.
A simple fix for this is
```
val showBottomSheetScaffold = rememberSaveable { isLayerRevealAnimationEnded }
```